site/en/blog/cascade-layers/index.md

```diff
@@ -0,0 +1,253 @@
+
+```css
+/* Base */
+@import '../styles/base/normalize.css' layer(base); /* normalize or rest file */
+@import '../styles/base/base.css' layer(base); /* body and base styles */
+@import '../styles/base/theme.css' layer(theme); /* theme variables */
+@import '../styles/base/typography.css' layer(theme); /* theme typography */
+@import '../styles/base/utilities.css' layer(utilities); /* base utilities */
+
+/* Layouts */
+@import '../styles/components/post.css' layer(layouts); /* post layout */
+
+/* Components */
+@import '../styles/components/cards.css' layer(components); /* imports card */
+@import '../styles/components/footer.css' layer(components); /* footer component */
+```
+
+The above code snippet has three layers: `base`,`layouts`, and `components`. The normalize, theme, and typography files in `base`, with a `post` file in `layouts`, and `cards` and `footer` both in `components`. On import of the file, the layers are instantiated using the layer function. An alternative approach would be to organize your layers at the top of the file, declaring them before any imports:
+
+```css
+@layer base,
+       theme,
+       layouts,
+       components,
+       utilities;
+```
+
+Now, the order in which you `@import` your styles won’t matter to the layer order, since it’s already established at the first instance of the layer name. That's one less thing to worry about. You can still set imported files to specific layers, but the ordering is already established.
+
+<figure>
+    {% Img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/CSUb8jZLi4QJUdmTM109.png", alt="Screenshot from Codepen Project", width="800", height="556" %}
+    <figcaption>Explore the <a href="https://codepen.io/web-dot-dev/project/editor/ZGQLkq">project on Codepen.</a>
+    </figcaption>
+</figure>
+
+{% Aside %}
+Importing styles in a CSS file like this chain as they load, so it's more performant to include everything in a `style` tag within your HTML. Stylesheets included via `link` are the most performant, but don't currently have the ability to load as layers. This is currently in discussion with the WHATWG.
+{% endAside %}
+
+
+## Layers and the cascade
+
+Let’s take a step back and see where layers are used as it relates to the wider cascade:
+
+{% Img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/jH8ydtcwmHg4rHTfqvRU.png", alt="Cascade Illustration", width="800", height="393" %}
+
+**UA normal < Local User @layer < Local User normal < Author @layers < Author normal < Author !important < Author @layer !important < Local User !important < UA !important**
+
+You may notice here that `@layer !important` styles are inverted. Instead of being *less* specific than non-layered (normal) styles, they become *more* specific. This is because of how `!important` works in the cascade: it breaks the normal cascading in your stylesheets and reverses the normal specificity values.
```

Comment:
```suggestion
You may notice here that `@layer !important` styles are inverted. Instead of being less specific than non-layered (normal) styles, they become more specific. This is because of how `!important` works in the cascade: it breaks the normal cascading in your stylesheets and reverses the normal specificity values.
```